### PR TITLE
Add qTrade integration

### LINKED ISSUE #1030: Implement qTrade Exchange
qtrade.io
https://icook.github.io/qtrade-docs/

### PR DESCRIPTION
**Pull request by mattkimmel**
- What is the purpose of this Pull Request?
Add support for the qTrade exchange.
- What is the related issue for this Pull Request (if this PR fixes issue, prepend with "Fixes" or "Closes")?
Closes #1030 
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

File: lib/cryptoexchange/exchanges/qtrade/services/pairs.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          pairs.each do |pair|
+            base = pair['market_currency']
+            target = pair['base_currency'] # This is correct, but confusing
+            market_pair = Cryptoexchange::Models::MarketPair.new
+            market_pair.base = base
+            market_pair.target = target
+            market_pair.market = Qtrade::Market::NAME
+            market_pairs << market_pair
+          end
+          pp market_pairs
```
Comment (wongy91) [inline lib/cryptoexchange/exchanges/qtrade/services/pairs.rb]: this is failing the gem.

Comment (mattkimmel) [reply lib/cryptoexchange/exchanges/qtrade/services/pairs.rb]: Sorry about that! Fixed in 98762b8.

**Comment by mattkimmel** [comment]
Fixed, and included specs that were missing from my original commit for some reason.

**Comment by wongy91** [comment]
LGTM.

**Pull request merged**